package.json

```diff
@@ -66,5 +67,10 @@
   },
   "release": {
     "extends": "@stoplight/scripts/release"
+  },
+  "prettier": {
```

Comment:
IIRC it does nothing and you'd need to specify these options in `tslint.json` for `prettier` rule.